models/Comment.php

```diff
@@ -84,7 +120,7 @@ function delete_comment(){
     // Fonction permettant d'approuver un commentaire
     function approve_comment(){
         if (empty($_GET['approve']) || !ctype_digit($_GET['approve'])) {
-            die("Ho ! Fallait préciser le paramètre id en GET !");
+            echo("Ho ! Fallait préciser le paramètre id en GET !");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of echo language construct is discouraged.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5344015)